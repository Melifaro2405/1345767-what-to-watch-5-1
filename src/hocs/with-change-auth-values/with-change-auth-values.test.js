import React from "react";
import renderer from "react-test-renderer";
import PropTypes from "prop-types";
import withChangeAuthValues from "./with-change-auth-values";

const MockComponent = (props) => {
  const {children} = props;
  return (
    <div>
      {children}
    </div>
  );
};

MockComponent.propTypes = {
  children: PropTypes.oneOfType([
    PropTypes.arrayOf(PropTypes.node),
    PropTypes.node
  ]).isRequired,
};

const MockComponentWrapped = withChangeAuthValues(MockComponent);

it(`withChangeAuthValues is rendered correctly`, () => {
  const tree = renderer.create((
    <MockComponentWrapped>
      <React.Fragment />
    </MockComponentWrapped>
  )).toJSON();

  expect(tree).toMatchSnapshot();
});
