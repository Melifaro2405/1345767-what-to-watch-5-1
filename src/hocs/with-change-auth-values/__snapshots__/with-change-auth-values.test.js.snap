// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withChangeAuthValues is rendered correctly 1`] = `<div />`;
